src/libs/OptionsListUtils.ts

```diff
@@ -204,6 +204,7 @@ type GetOptionsConfig = {
     recentAttendees?: Attendee[];
     shouldSeparateWorkspaceChat?: boolean;
     shouldSeparateSelfDMChat?: boolean;
+    excludeHiddenReports?: boolean;
```

Comment:
@luacmartins Possible to add a unit test for the flag?

Reply:
Updated tests to cover this case